src/main/kotlin/no/nav/klage/oppgave/service/KlagebehandlingService.kt

```diff
@@ -132,6 +124,47 @@ class KlagebehandlingService(
         return klagebehandling
     }
 
+    fun createKlagebehandlingFromMottak(mottak: Mottak) {
+        if (klagebehandlingRepository.findByMottakId(mottak.id) != null) {
+            logger.error("We already have a klagebehandling for mottak ${mottak.id}, will not create a new one, nor update the existing one.")
```

Comment:
Burde vi ikke kaste exception her?

Reply:
Jeg er litt usikker på om dette skal kunne inntreffe eller ikke. Koden for mottak (oversendelse) er ganske ufullstendig fremdeles. Vi mottar en UUID i OversendtKlage, men bruker den ikke til noe. Så sånn koden er nå, så vil vi opprette et nytt mottak hver gang vi for en oversendelse, selv om det er på samme UUID. Det er vel ikke meningen, vil jeg tro?
Så mulig vi kan, eller bør, kaste en exception, men svaret mitt er vel at vi burde fintune OversendelsesService før vi gjør noe mer i KlagebehandlingService, for da vil vi vite mer om hvorvidt det er en feilsituasjon eller ikke og hvordan det bør behandles. Jeg tror det viktige i første omgang bare er at vi faktisk får laget Klagebehandlinger.

Reply:
Er nok mye mer riktig å kaste exception med en gang vi mottar samme eksterne referanse ja.